### PR TITLE
Feature/te 478 fix error for mistyped parameters

### PR DESCRIPTION
**Pull request by gunther-bachmann**

### CODE REVIEW

File: aml/org.testeditor.aml.dsl.tests/src/org/testeditor/aml/dsl/tests/common/AmlTestModels.xtend [hunk 1]
```diff
@@ -41,6 +41,12 @@ class AmlTestModels {
 			]					
 			interactionTypes += interactionTypeForWait
 
+			val interactionTypeIntoAndWait = interactionType("typeIntoAndWait") => [
+				defaultMethod = methodReference(resourceSet, DummyFixture, "typeIntoAndWait", "element", "locatorStrategy", "value", "seconds")
+				template = template("type").withParameter(defaultMethod.parameters.findFirst[name=='value']).withText("into").withParameter("element").withText("and wait").withParameter(defaultMethod.parameters.findFirst[name=='seconds'])
```
Comment (kpanier) [inline aml/org.testeditor.aml.dsl.tests/src/org/testeditor/aml/dsl/tests/common/AmlTestModels.xtend]: Format?

File: common/org.testeditor.dsl.common.ui.tests/src/org/testeditor/dsl/common/ui/utils/ProjectContentGeneratorTest.xtend [hunk 2]
```diff
@@ -157,6 +159,7 @@ class ProjectContentGeneratorTest extends AbstractTest {
 	@Test
 	def void testGetVersion() {
 		// given (this is a plugin test)
+		Assume.assumeTrue(isWithinOSGIContext)
```
Comment (kpanier) [inline common/org.testeditor.dsl.common.ui.tests/src/org/testeditor/dsl/common/ui/utils/ProjectContentGeneratorTest.xtend]: Why not a static import?

Comment (gunther-bachmann) [inline common/org.testeditor.dsl.common.ui.tests/src/org/testeditor/dsl/common/ui/utils/ProjectContentGeneratorTest.xtend]: Static import of what exactly? I'm not sure what you mean. Could you please write some more on this?

Comment (kpanier) [inline common/org.testeditor.dsl.common.ui.tests/src/org/testeditor/dsl/common/ui/utils/ProjectContentGeneratorTest.xtend]: static import for Assume.assumeTrue

Comment (gunther-bachmann) [inline common/org.testeditor.dsl.common.ui.tests/src/org/testeditor/dsl/common/ui/utils/ProjectContentGeneratorTest.xtend]: ahh, no problem

**Pull request merged**